crates/nu-command/tests/commands/rename.rs

```diff
@@ -1,116 +1,85 @@
-    Playground::setup("rename_test_2", |dirs, sandbox| {
-        sandbox.with_files(vec![FileWithContentToBeTrimmed(
-            "los_cuatro_mosqueteros.txt",
-            r#"
-                Andrés N. Robalino
-                JT Turner
-                Yehuda Katz
-                Jason Gedge
-            "#,
-        )]);
+    let sample = r#"
+            [["Andrés N. Robalino"],
+             ["JT Turner"],
+             ["Yehuda Katz"],
+             ["Jason Gedge"]]
+            "#;
 
-        let actual = nu!(
-            cwd: dirs.test(), pipeline(
-            r#"
-                open los_cuatro_mosqueteros.txt
-                | lines
+    let actual = nu!(pipeline(&format!(
+        r#"
+                {sample}
                 | wrap name
                 | default "arepa!" hit
                 | rename mosqueteros
                 | get hit
                 | length
                 "#
-        ));
+    )));
 
-        assert_eq!(actual.out, "4");
-    })
+    assert_eq!(actual.out, "4");
 }
 
 #[test]
 fn errors_if_no_columns_present() {
-    Playground::setup("rename_test_3", |dirs, sandbox| {
-        sandbox.with_files(vec![FileWithContentToBeTrimmed(
-            "los_cuatro_mosqueteros.txt",
-            r#"
-                Andrés N. Robalino
-                JT Turner
-                Yehuda Katz
-                Jason Gedge
-            "#,
-        )]);
+    let sample = r#"
+            [["Andrés N. Robalino"],
+             ["JT Turner"],
+             ["Yehuda Katz"],
+             ["Jason Gedge"]]
+            "#;
 
-        let actual = nu!(
-            cwd: dirs.test(), pipeline(
-            "
-                open los_cuatro_mosqueteros.txt
-                | lines
+    let actual = nu!(pipeline(&format!(
+        "
+                {sample}
                 | rename mosqueteros
                 "
-        ));
+    )));
 
-        assert!(actual.err.contains("command doesn't support"));
-    })
+    assert!(actual.err.contains("command doesn't support"));
 }
 
 #[test]
 fn errors_if_columns_param_is_empty() {
-    Playground::setup("rename_test_4", |dirs, sandbox| {
-        sandbox.with_files(vec![FileWithContentToBeTrimmed(
-            "los_cuatro_mosqueteros.txt",
-            r#"
-                Andrés N. Robalino
-                JT Turner
-                Yehuda Katz
-                Jason Gedge
-            "#,
-        )]);
+    let sample = r#"
+            [["Andrés N. Robalino"],
+             ["JT Turner"],
+             ["Yehuda Katz"],
+             ["Jason Gedge"]]
+            "#;
 
-        let actual = nu!(
-            cwd: dirs.test(), pipeline(
-            r#"
-                open los_cuatro_mosqueteros.txt
-                | lines
+    let actual = nu!(pipeline(&format!(
+        r#"
+                {sample}
                 | wrap name
                 | default "arepa!" hit
-                | rename --column {}
+                | rename --column {{}}
```

Comment:
Maybe it makes sense here to simply move the sample string into the pipe so we don't need the double `{{}}` here but that is just a stylistic minutia.

Reply:
I believe it is better to keep the formatting inside the `pipeline`, as the sample data contains multiple lines, and `nu!` treats all lines in its argument as separate commands.

Reply:
Ah right!